webgme/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I'm wary about including any wildcard import declarations in DefinitelyTyped at all. If `webgme` itself doesn't define a loader for `text!*` imports, that shouldn't be part of the typings. The tests could just use `const MetaDataStr = "";`.